Index: furo/src/furo/assets/styles/base/_screen-readers.sass
===================================================================
--- furo.orig/src/furo/assets/styles/base/_screen-readers.sass
+++ furo/src/furo/assets/styles/base/_screen-readers.sass
@@ -9,5 +9,5 @@
   white-space: nowrap !important
   border: 0 !important
 
-:-moz-focusring
+\:-moz-focusring
   outline: auto
Index: furo/src/furo/assets/styles/content/_target.sass
===================================================================
--- furo.orig/src/furo/assets/styles/content/_target.sass
+++ furo/src/furo/assets/styles/content/_target.sass
@@ -1,4 +1,4 @@
-:target
+\:target
   scroll-margin-top: 0.5rem
 
 @media (max-width: $full-width - $sidebar-width)
@@ -41,7 +41,7 @@ figure:hover > figcaption > p > .headerl
 table:hover > caption > .headerlink
   visibility: visible
 
-:target >,  // Regular section[id] style anchors
+\:target >,  // Regular section[id] style anchors
 span:target ~ // Non-regular span[id] style "extra" anchors
   h1,
   h2,
Index: furo/src/furo/assets/styles/extensions/_sphinx-design.sass
===================================================================
--- furo.orig/src/furo/assets/styles/extensions/_sphinx-design.sass
+++ furo/src/furo/assets/styles/extensions/_sphinx-design.sass
@@ -30,7 +30,7 @@ body
   box-shadow: 0 -2px var(--sd-color-tabs-overline), 0 1px var(--sd-color-tabs-underline)
 
 // Shadows
-.sd-card  // Have a shadow by default
+.sd-card
   box-shadow: 0 0.1rem 0.25rem var(--sd-color-shadow), 0 0 0.0625rem rgba(0, 0, 0, 0.1)
 
 .sd-shadow-sm
@@ -46,6 +46,6 @@ body
 .sd-card-hover:hover  // Don't change scale on hover
   transform: none
 
-.sd-cards-carousel  // Have a bit of gap in the carousel by default
+.sd-cards-carousel
   gap: 0.25rem
   padding: 0.25rem
Index: furo/src/furo/assets/styles/_scaffold.sass
===================================================================
--- furo.orig/src/furo/assets/styles/_scaffold.sass
+++ furo/src/furo/assets/styles/_scaffold.sass
@@ -110,7 +110,7 @@ article
 .toc-sticky
   position: sticky
   top: 0
-  height: min(100%, 100vh)
+  //height: min(100%, 100vh)
   height: 100vh
 
   display: flex
